Zappy_IA_Report/NN/Code/Tutorial5.py

```
@@ -0,0 +1,45 @@
+
+word_index = data.get_word_index()
+
+word_index = {k:(v+3) for k, v in word_index.items()}
+word_index["<PAD>"] = 0
+word_index["<START>"] = 1
+word_index["<UNK>"] = 2
+word_index["<UNUSED>"] = 3
+
+reverse_word_index = dict([(value, key) for (key, value) in word_index.items()])
+
+def decode_review(text):
```

Ruff (0.11.9)

Missing return type annotation for public function `decode_review`

(ANN201)